src/Dialog/ConfirmationDialog.jsx

```diff
@@ -34,13 +34,20 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import React from 'react';
+import './confirmation-dialog.scss';
```

Comment:
In other files we put the SCSS imports always last. Let us stick to that convention.

(We could also change the convention to always put the SCSS imports first.)

Reply:
![image](https://user-images.githubusercontent.com/4866953/89888873-fa76ed80-dbd0-11ea-9926-34e80f25f004.png)
I just use the `sort-imports` extension with its default settings to sort all imports. It's very convenient. I would prefer to use it to make the order consistent. Do you think if that is ok? I can update the other files with the extension if it is ok. Or I can just move the single line.

Reply:
ok, since I removed .scss file, the import is not needed any more.

Reply:
The extension looks nice; even better would be if we use an ESLint plugin for it, since then it would be checked on the build server and everybody could use it regardless of the IDE. [eslint-plugin-simple-import-sort](https://github.com/lydell/eslint-plugin-simple-import-sort) looks like such a plugin, in case you want to look further into that.